pkg/subctl/cmd/join.go

```diff
@@ -163,6 +166,12 @@ func joinSubmarinerCluster(subctlData *datafile.SubctlData) {
 		err = lighthouse.Ensure(status, config, "", "", false)
 		status.End(err == nil)
 		exitOnError("Error deploying multi cluster service discovery", err)
+
+		status.Start("Joining to Kubefed control plane")
+		cmd := exec.Command("kubefedctl", "join", "--kubefed-namespace", "kubefed-operator", clusterID, "--host-cluster-context", kubefedController)
```

Comment:
You can use something like:
`_, err := exec.Command("kubefedctl", "join", "--kubefed-namespace", "kubefed-operator", clusterID, "--host-cluster-context", kubefedController).Output()

Since you're running single command, you don't really need CombinedOutput

Reply:
Doesn't that look little hard to read, since the entire command is a set of strings than variable?

Reply:
It is same thing, just combining Command and CombinedOutput() in single line.

Reply:
Done.